apps/studio/src/theme/foundations/colours.ts

```diff
@@ -0,0 +1,7 @@
+export const colours = {
+  base: {
+    canvas: {
+      light: '#babecb',
```

Comment:
sudden addition of color tokens? can you[ generate isomer's theme](https://github.com/opengovsg/design-system/tree/main/token-gen) from figma tokens instead?
or does not exist yet?

Reply:
the colour tokens doesn't exist yet - we're just piggy backing off default theme. i added this so that we can use colour tokens now but when we put our actual tokens inside, it'll be correct also